AdminWebsite/AdminWebsite/ClientApp/src/app/security/guards/last-minute-amendments.guard.spec.ts

```diff
@@ -63,39 +58,4 @@ describe('LastMinuteAmendmentsGuard', () => {
             expect(router.navigate).toHaveBeenCalledTimes(1);
         });
     });
-
-    describe('when accessing assign-judge; last minute', () => {
-        it('ejudFeature flag off should override last-minute-amendment-guard and block assign-judge url to be reach', () => {
-            // setup
-            guard.isV2 = false;
-            const url = 'assign-judge';
-            const dataSnapshot = { exceptionToRuleCheck: true } as Data;
-            const urlSegmentArray = [{ path: url }] as UrlSegment[];
-            // execute
-            const returned = guard.canActivate(
-                <ActivatedRouteSnapshot>{ url: urlSegmentArray, data: dataSnapshot },
-                <RouterStateSnapshot>{ url: url }
-            );
-            // assert
-            expect(returned).toBe(false);
-            expect(router.navigate).toHaveBeenCalledWith([redirectPath]);
-        });
-
-        it('ejudFeature flag on should allow last-minute-amendment-guard and allow assign-judge url to be reached', () => {
```

Comment:
please keep the scenario of ejud flag on

Reply:
reinstated